README.md

```diff
@@ -617,6 +617,7 @@ User can add a STM32 based board following this [wiki](https://github.com/stm32d
 | :green_heart:  | STM32L041G6 | Generic Board | *2.4.0* |  |
 | :green_heart: | STM32L041K6T | Generic Board | *2.0.0* |  |
 | :green_heart: | STM32L051C6<br>STM32L051C8 | Generic Board | *2.0.0* |  |
+| :green_heart: | STM32L051K8 | Generic Board | *2.8.0* |  |
```

Comment:
As well the  L051K6Tx and L051K6Ux should also be referenced.

Reply:
Done.